src/common/runtime/cmdline.ts

```diff
@@ -49,6 +51,11 @@ for (let i = 0; i < sys.args.length; ++i) {
     } else if (a === '--expectations') {
       const expectationsFile = new URL(sys.args[++i], `file://${sys.cwd()}`).pathname;
       loadWebGPUExpectations = import(expectationsFile).then(m => m.expectations);
+    } else if (a === '--gpu-provider') {
+      const modulePath = sys.args[++i];
+      setGPUProvider(() => {
+        return require(modulePath).gpu;
+      });
```

Comment:
nit: could be more concise with:
```suggestion
      setGPUProvider(() => require(modulePath).gpu);
```

Reply:
Done